new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java

```diff
@@ -275,7 +340,11 @@ private static void createMiiDomainWithMultiClusters() {
                     .name("USER_MEM_ARGS")
                     .value("-Djava.security.egd=file:/dev/./urandom ")))
             .adminServer(new AdminServer()
-                .serverStartState("RUNNING"))
+                .serverStartState("RUNNING")
+                .adminService(new AdminService()
+                    .addChannelsItem(new Channel()
+                        .channelName("default")
+                        .nodePort(0))))
```

Comment:
Does this work to use 0 for the nodePort? My expectation is that Kubernetes will fill in the port on the NodePort-type Service. But the introspector won't know what port will be filled in...  Or, asked another way, this might work for HTTP, but won't work for T3.

Reply:
This is not for t3. It is not used in the test. It is just for debugging purpose for me to connect to the admin console. I can remove this channel.